lib/matplotlib/pyplot.py

```diff
@@ -15,7 +16,19 @@
     y = np.sin(x)
     plt.plot(x, y)
 
-The object-oriented API is recommended for more complex plots.
+The object-oriented API is recommended for complex plots, though pyplot
+is still usually used to create the figure and often the axes in the figure.
+See `.pyplot.figure`, `.pyplot.subplots`, and `.pyplot.subplot_mosaic` to
+create figures, and :doc:`Axes API <doc/api/axes_api>` for the plotting
```

Comment:
Do any sphinx experts understand how I am to reference this from here?

Reply:
```suggestion
create figures, and :ref:`Axes API <maplotlib.axes.Axes>` for the plotting
```

I think this will work?